src/__integ__/__snapshots__/themes.test.ts.snap

```diff
@@ -28,8 +28,8 @@ Object {
   "border-radius-code-editor": "0px",
   "border-radius-container": "0px",
   "border-radius-control": "2px",
-  "border-radius-control-focus-ring-circle": "50%",
-  "border-radius-control-focus-ring-default": "2px",
+  "border-radius-control-circular-focus-ring": "50%",
```

Comment:
Do we have circular controls other than radio input? I mean, `border-radius-control-radio-focus-ring` can be more telling.

Reply:
Yes, we use the same token in the hotspot